### ISSUE TITLE
Recaptcha is still being showed

### ISSUE THREAD

**Issue by Ulixestoitaca** [Francesco149/fuckcaptcha#2]
I have run the code but nothing happens.
No operation is done, no error is shown. What is it supposed to do exactly?

Recaptcha is showing puzzle once clicked on checkbox so Chromium is still traced as an automated browser

**Comment by Francesco149** [comment]
yes, it should show the puzzle. this bypasses detections that will make the puzzle always fail even if you get it right. it should be used with a captcha solver such as nonoCAPTCHA

**Comment by Ulixestoitaca** [comment]
Thank you for your response first of all. I just would need to avoid chromium to be detected by recaptcha (if I use chromium not driven by pyppeteers recaptcha is not shown) . Do u have any idea how to do this by pyppeteer? Il 31 ago 2020 6:54 PM, "Franc[e]sco" <notifications@github.com> ha scritto:
yes, it should show the puzzle. this bypasses detections that will make the puzzle always fail even if you get it right. it should be used with a captcha solver such as nonoCAPTCHA

**Comment by Francesco149** [comment]
I don't know of any solutions that can do that. Current captcha solvers use this kind of bypass for detections, then automatically solve the puzzle (or use voice recognition) with pyppetteer
